### DISCUSSION TITLE
[BETA] Acrylic Support and more

### DISCUSSION THREAD

**Discussion by cryptofyre** [ciderapp/Apple-Music-Electron#172] [General]
Unfinished so make sure to report feedback with the [BETA] flare.

<hr /><em>This discussion was created from the release <a href='https://github.com/Apple-Music-Electron/Apple-Music-Electron/releases/tag/v2.5.0'>[BETA] Acrylic Support and more</a>.</em>

**Comment by Nosh118** [comment]
pretty swag can't lie

**Comment by tenhyperfuel** [comment]
I have weird issues all over, is this expected in the beta (pretty much un-usable)? 2.41 doesn't work either :(

**Comment by tenhyperfuel** [comment]
### [BETA]
![image](https://user-images.githubusercontent.com/78945267/130351582-3111b19a-17d2-4caa-a016-baa23dbb7bb3.png)
 restarting doesn't fix it

**Reply by coredev-uk** [reply]
Do a clean install

**Reply by tenhyperfuel** [reply]
that was my first time using that version, going back to 2.41 and enabling auth mode solved it, the client is very much usable now, thanks anyway :D

**Comment by ghost** [comment]
For me, when I login, it doesn't login properly. I mean it shows me everything but when I quit it just asks me to login again. Anyone know what this is?

**Reply by tenhyperfuel** [reply]
which version are you using

**Reply by ghost** [reply]
2.5.0 Beta
